config/validated_config_db_connector.py

```diff
@@ -0,0 +1,37 @@
+import jsonpatch
+from jsonpointer import JsonPointer
+
+from generic_config_updater.generic_updater import GenericUpdater, ConfigFormat
+
+def ValidatedConfigDBConnector(config_db_connector):
```

Comment:
As we discussed on design, we need to check global disable/enable option. So suggest add the check in this function.

Reply:
Sorry I did not mean `fallback_adhoc_validation`. I mean `yang_config_validation` you added in ConfigDB. You may check @wen587 PR [https://github.com/sonic-net/sonic-buildimage/pull/11715/files](https://github.com/sonic-net/sonic-buildimage/pull/11715/files)

Reply:
`fallback_adhoc_validation` should be independent with `ValidatedConfigDBConnector`. We have use cases like
1. True, True
2. True, False
3. False, True
4. False, False
So just remove this parameter in this class.

Reply:
Still applicable.

Reply:
I changed to make `fallback_adhoc_validation` independent of `ValidatedConfigDBConnector`. In `ValidatedConfigDBConnector`, we use `yang_config_enabled`